test/e2e/storage/utils/host_exec.go

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
We are in 2019. I don't know what the policy is for updating existing files, but at least for new ones I'd expect 2019 here.